content/en/docs/about/membership.md

```diff
@@ -0,0 +1,193 @@
+- Have read the [contributor guide]
+- Sponsored by 2 reviewers. **Note the following requirements for sponsors**:
+- **[Open an issue][membership request] against the kubeflow/internal-acls repo**
+  - Ensure your sponsors are @mentioned on the issue
+- **Open a pull request against the kubeflow/internal-acls repo**
+  - Complete every item on the checklist ([preview the current version of the template][membership template])
+  - Make sure that the list of contributions included is representative of your work on the project.
+- Have your sponsoring reviewers reply confirmation of sponsorship
+- Once your sponsors have responded, your request will be reviewed by the Kubeflow team. Any missing information will be requested
+
+### Responsibilities
+
+- Responsive to issues and PRs assigned to them
+- Responsive to mentions of teams they are members of
+- Active owner of code they have contributed (unless ownership is explicitly transferred)
+  - Code is well tested
+  - Tests consistently pass
+  - Addresses bugs or issues discovered after code is accepted
+- Subscribed to <kubeflow-discuss@googlegroups.com>
+
+**Note:** members who frequently contribute code are expected to proactively perform code reviews and work towards becoming a primary *reviewer* for the subproject that they are active in.
+
+### Privileges
+
+- Members can do `/lgtm` on open PRs.
+- They can be assigned to issues and PRs, and people can ask members for reviews with a `/cc @username`.
+- Tests can be run against their PRs automatically. No `/ok-to-test` needed.
+- Members can do `/ok-to-test` for PRs that have a `needs-ok-to-test` label, and use commands like `/close` to close PRs as well.
+
+## Reviewer
+
+Reviewers are able to review code for quality and correctness on some part of a subproject. They are knowledgeable about both the codebase and software engineering principles.
+
+**Defined by:** *reviewers* entry in an OWNERS file in a repo owned by the Kubeflow project.
+
+Reviewer status can be scoped to either parts of the codebase or the root directory for the entire codebase
+
+**Note:** Acceptance of code contributions requires at least one approver in addition to the assigned reviewers.
+
+### Requirements
+
+The following apply to the part of codebase for which one would be a reviewer in an [OWNERS] file.
+
+- member for at least 3 months
+- Primary reviewer for at least 5 PRs to the codebase
+- Reviewed or merged at least 20 substantial PRs to the codebase
+- Knowledgeable about the codebase
+- Sponsored by a subproject approver
+  - With no objections from other approvers
+  - Done through PR to update the OWNERS file
+- May either self-nominate or be nominated by an approver in this subproject
+
+**Note:** Working Group Leads may nominate and approve `Reviewers` that don't meet these requirements due to exceptional circumstances. While acceptable in the short term, Working Group Leads should ensure that these `Reviewers` eventually meet the requirements
```

Comment:
Should we numerate these exception ?

Reply:
I'm not actually aware of specific exceptions, this is more so in place if some of the existing projects have reviewers/owners that don't meet the bar _yet_. I'm sure there has usually been a good reason for a reviewer/approver to be appointed when not meeting the requirements.

I expect the need for exceptions to go down as the community grows

Reply:
>I'm not actually aware of specific exceptions, this is more so in place if some of the existing projects have reviewers/owners that don't meet the bar yet. I'm sure there has usually been a good reason for a reviewer/approver to be appointed when not meeting the requirements.

Should we follow the rules in that case ? I feel that we should either cleanup those OWNERs files or enumerate the exceptions. WDYT @franciscojavierarceo @kubeflow/kubeflow-steering-committee ?

Reply:
I didn't want to disrupt the existing projects since these guidelines haven't explicitly been stated before. I'd rather we introduce a timeline (say, 3 months) for all projects to audit and clean up their OWNERS files. That being said, I'll defer to the KSC on this!

Reply:
I don't have a preference here and don't think it's a blocker for this PR IMO.

Reply:
I would suggest to remove this Note from the doc for now. We can always update it once we discuss those exceptions.
What do you think @terrytangyuan @anishasthana ?